### ISSUE TITLE
BUG Fatal Exception: ej: lateinit property manager has not been initialized

### ISSUE THREAD

**Issue by thinhbuzz** [azhon/AppUpdate#141]
**Bug标题**
清楚简明的描述你遇到的问题

**AppUpdate参数设置**
![image](https://user-images.githubusercontent.com/8065692/205936257-6159c9be-f43f-40d3-bfec-06619d61c87f.png)

**复现步骤**

**错误Logcat日志截图**

**软件版本信息**
 - Android Studio：Android Studio Dolphin | 2021.3.1 Patch 1
 - AppUpdate：4.2.4
 - 设备品牌：it happens on multiple devices, i only get the error through firebase
 - 设备系统版本：android 10 11 and 12

**额外有用的信息**
Firebase crash log
```text
Fatal Exception: ej: lateinit property manager has not been initialized
       at com.azhon.appupdate.view.UpdateDialogActivity.onBackPressed(UpdateDialogActivity.java:16)
       at android.app.Activity.onKeyUp(Activity.java:3884)
       at android.view.KeyEvent.dispatch(KeyEvent.java:2891)
       at android.app.Activity.dispatchKeyEvent(Activity.java:4190)
       at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java)
       at androidx.core.view.KeyEventDispatcher.b(KeyEventDispatcher.java:11)
       at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:18)
       at androidx.appcompat.app.AppCompatActivity.dispatchKeyEvent(AppCompatActivity.java:13)
       at androidx.appcompat.view.WindowCallbackWrapper.dispatchKeyEvent(WindowCallbackWrapper.java:2)
       at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.dispatchKeyEvent(AppCompatDelegateImpl.java:20)
       at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:403)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:6268)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:6131)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5620)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5677)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5643)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5795)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5651)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5852)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5624)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5677)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5643)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5651)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5624)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5677)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5643)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5828)
       at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5992)
       at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3184)
       at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2726)
       at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2717)
       at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3161)
       at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:335)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:8051)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:620)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1011)
```

**Comment by azhon** [comment]
Usually, this bug will not occur. It seems that the`manager: DownloadManager` needs to be changed to be nullable.

**Comment by azhon** [comment]
Fixed on version 4.2.6
